packages/react/src/primitives/View/View.tsx

```diff
@@ -29,11 +26,8 @@ const ViewInner = <Element extends ElementType = 'div'>(
     as,
     {
       'aria-label': ariaLabel,
-      className,
       'data-testid': testId,
       disabled: isDisabled,
-      id,
-      role,
```

Comment:
This is a bug. It basically provides `undefined` props and if we render like this `<View as={RadixTab}  />` but not supplying `role` and `id`, they will be overridden and clear out, probably through `{...rest}` internally.

Reply:
So, `className`, `role` and `id` will be passed through the `{...nonStyleProps}` correct?

Reply:
Yes, exactly. But if we list it explicitly and not specify values for them, then it pass in `undefined`

Reply:
Yeah, I've seen a related issue with the Radix `Dropdown` primitives we use in `Menu` where a passed `className` prop overrode the internal `className` and would break the component behavior. It looks like we have a test around `id`, and `role` but not for `className`. Would you mind adding a `className` test to `View.test.tsx`?